css/webots-doc.css

```diff
@@ -17,6 +17,9 @@
   src: local("fonts/Roboto-Mono"), url("fonts/Roboto-Mono.ttf") format("truetype");
 }
 
+b, strong {
```

Comment:
Is this still required?

Reply:
It's better practice, but it's technically not needed, so I removed it.